pilot/pkg/features/ambient.go

```diff
@@ -44,6 +44,11 @@ var (
 		"PILOT_PREFER_SENDING_HBONE",
 		false, false,
 		"If enabled, HBONE will be preferred when sending to destinations. ")
+
+	DefaultAllowFromWaypoint = registerAmbient(
+		"PILOT_DEFAULT_ENFORCE_FROM_WAYPOINT",
```

Comment:
What does "DEFAULT" mean ? Or "ENFORCE" ? 

PILOT_ALLOW_FROM_WAYPOINT ? It doesn't seem a default, and doesn't enforce but allows.

Reply:
DEFAULT was just copying part of the name 

ALLOW *with no other policies) effectively denies everything that doesn't match. Open to bikeshedding names.